### ISSUE TITLE
Add a class related to the IMF's instantaneous frequency

### ISSUE THREAD

**Issue by YihaoSu** [HHTpy/HHTpywrapper#2]
This issue is to add a class for calculating the instantaneous frequency and amplitude of a IMF.

**Comment by YihaoSu** [comment]
I have added EEMD class in eemd.py

**Comment by ZhaoShaw** [comment]
Hi，
I have some questions about the example you give. Does the variables iamp and ifreq represent the instantaneous amplitude (or instantaneous phase?) , and the instantaneous frequency of the signal (**structure signal, Z(t)=X(t)+iY(t)**) ?

**Comment by YihaoSu** [comment]
Yes. The instance of [class HSA](https://github.com/HHTpy/HHTpywrapper/blob/master/hhtpywrapper/hsa.py#L6) has two attributes (variables): iamp (instantaneous amplitude of a signal) and ifreq (instantaneous frequency of a signal). This class calls the MATLAB function 'fa' to calculate these variables. You can find the source code of 'fa' function in /Path to your Python site-packages/hhtpywrapper-0.1.dev0-py3.6.egg/hhtpywrapper/HHT_MATLAB_package/HSA/fa.m

You can read the paper '[On instantaneous frequency](http://rcada.ncu.edu.tw/ref/reference005.pdf)' and the [doc of MATLAB hilbert function](http://www.mathworks.com/help/signal/ref/hilbert.html) to understand how the 'fa' function calculate instantaneous amplitudes and frequencies .

**Comment by ZhaoShaw** [comment]
Thank you for your answer!
I have read the matlab source code but I have not been able to read hilbert.m carefully before. Meanwhile, thank you for your recommendation, I will read it with my free time!
